imgparse/xmp.py

```diff
@@ -49,6 +49,21 @@ class DJITags(SensorTags):
     YAW = "drone-dji:GimbalYawDegree"
     FOCAL_LEN = "drone-dji:CalibratedFocalLength"
     SELF_DATA = "drone-dji:SelfData"
+    WAVELENGTH_CENTRAL = "Camera:CentralWavelength"
+    WAVELENGTH_FWHM = "Camera:WavelengthFWHM"
+    BANDNAME = "Camera:BandName"
```

Comment:
Same here. Do you have a sample DJI multispectral image we can add to the test set to test these tags?

Reply:
https://github.com/SenteraLLC/py-image-metadata-parser/pull/63/commits/128f96f0bb5a2d923c00bdf2fdef61f780c3febb Added and added tests